src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Figure implements Area, Output{
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The 'Area' and 'Output' interfaces should not be implemented together if their methods are not conceptually related.